lib/location/city.ex

```diff
@@ -22,6 +28,13 @@ defmodule Location.City do
     end
   end
 
+  def get_city(name, country_code) do
```

Comment:
I haven't added tests yet because I wanted to validate this experiment with you first. See